src/planner/operator/schema.cpp

```diff
@@ -18,6 +18,7 @@ f_group_pos Schema::createGroup() {
 void Schema::insertToScope(const std::shared_ptr<Expression>& expression, f_group_pos groupPos) {
     KU_ASSERT(!expressionNameToGroupPos.contains(expression->getUniqueName()));
     expressionNameToGroupPos.insert({expression->getUniqueName(), groupPos});
+    KU_ASSERT(getGroup(groupPos)->expressionNameToPos.contains(expression->getUniqueName()));
```

Comment:
I wouldn't assert this. Instead the assertion already exist in `FactorizationGroup::getExpressionPos`

Reply:
hmm I think the assertion in `FactorizationGroup::getExpressionPos` serves for a different purpose from this one.